org.eclipse.xtext.xbase/model/Xbase.ecore

```diff
@@ -277,7 +279,7 @@
         containment="true"/>
     <eStructuralFeatures xsi:type="ecore:EReference" name="eachExpression" eType="#//XExpression"
         containment="true"/>
-    <eStructuralFeatures xsi:type="ecore:EReference" name="declaredParam" eType="ecore:EClass platform:/resource/org.eclipse.xtext.common.types/model/JavaVMTypes.ecore#//JvmFormalParameter"
+    <eStructuralFeatures xsi:type="ecore:EReference" name="declaredParam" eType="ecore:EClass ../../org.eclipse.xtext.common.types/model/JavaVMTypes.ecore#//JvmFormalParameter"
```

Comment:
is this change intended?

Reply:
Looks like a new EMF version did some fance optimization. Good catch.